src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readFile(fromFileName);
+        String report = generateReport(data);
+        writeToFile(toFileName, report);
+    }
+
+    private String generateReport(String data) {
+        StringBuilder report = new StringBuilder();
+        String[] stringArray = data.split(" ");
+        String supply = stringArray[0];
+        String buy = stringArray[1];
+        String result = stringArray[2];
```

Comment:
again, this is fragile approach
the fact that you return string,not int[] doesn't make it better, if you still need to parse it. Don't create custom formats that need to be encoded into and then decoded from. Make readFile do no business logic - only read from file, return string of content. Move all the logic to generate report method